scripts/util.js

```diff
@@ -13,24 +14,24 @@ util.toggleAboutMe = function() {
     $('.blogPosts').fadeIn();
   });
 };
-util.getTemplate = function (data) {
-  Article.prototype.compiled = Handlebars.compile(data);
-};
-util.templateLoaded = function() {
-  util.toggleAboutMe();
-  blog.render();
-  util.truncateArticles();
-  blog.showFilteredArts();
-};
-util.compileTemplate = function(){
-  $.get('templates/article-template.handlebars', util.getTemplate)
-    .done(util.templateLoaded);
-};
+//not sure this belongs here
 util.truncateArticles = function() {
-  $('.blogBody p:not(:first-child)').hide();
+  console.log('truncate');
+  // $('.blog-body').hide();
+  ///taken from demo and adapted to fit my code base.
+  $('.blog-body').children(':nth-child(n+5)').hide();
   $('main').on('click', '.readOn', function(event){
     event.preventDefault();
-    $(this).parent().find('p').fadeIn();
+    $(this).prev('.blog-body').children().fadeIn();
+    // $(this).parent().find('.blog-body').fadeIn();
     $(this).hide();
   });
 };
+//pretty sure this does belong here
```

Comment:
yes! util best for generic helpers that can be applied in a variety of places (like a date fetcher). the two you're unsure of above are fine if they work but might be better as blog functions, since they explicitly manage features of the blog.